bindings/python/htcondor2/_collector.py

```diff
@@ -41,31 +42,36 @@ class Collector():
     # In version 1, there was a distinct DaemonLocation type (a named tuple)
     # that `pool` could also be, but that functionality was never documented.
     #
-    def __init__(self, pool : Union[str, classad.ClassAd, List[str], Tuple[str], None] = None):
+    def __init__(self, pool : Union[str, classad.ClassAd, List[str], Tuple[str], None] = None, security : SecurityContext = None ):
         """
         :param pool:  A ``host::port`` string specifying the remote collector,
                       a list (or tuple) of such strings, or a ClassAd
                       with a ``MyAddress`` attribute (such as might be returned
                       by :meth:`locate`).  :py:obj:`None` means the value of the
                       configuration parameter ``COLLECTOR_HOST``.
+        :param security: SecurtiyContext to use for authenteication.
```

Comment:
```suggestion
        :param security: SecurityContext to use for authentication.
```
Spelling correction.  Also, I don't know if this will automatically be linked to the `SecurityContext` documentation; if it isn't, it should either be "Security context" or
```
class:`htcondor2.SecurityContext`
```
or the like.